app/templates/components/volunteer-headshot.hbs

```diff
@@ -0,0 +1,9 @@
+<img class="volunteer-headshot__image icon large" src={{volunteer.photoThumbUrl}} alt={{altText}} />
+{{#if isVolunteerNamePresent}}
+  <p class="volunteer-headshot__name" data-test-selector="volunteer name">{{volunteerName}}</p>
+  {{#if userRole}}
+    <p class="volunteer-headshot__role" data-test-selector="volunteer role">{{userRole.role.name}}</p>
+  {{else}}
+    <p class="volunteer-headshot__role" data-test-selector="volunteer role">Volunteer</p>
+  {{/if}}
```

Comment:
Rather than doing the `if/else` block, it looks like we can get away with an inline `if` since we are only updating 1 item.

```
<p class="volunteer-headshot__role" data-test-selector="volunteer role">
  {{if userRole userRole.role.name "Volunteer"}}
</p>
```